### PR TITLE
chore(core): factorize FftwError polynomial size check

### PR DESCRIPTION
**Pull request by IceTDrinker**
### Checklist 

(Use '[x]' to check the checkboxes, or submit the PR and then click the checkboxes)

* [ ] Tests for the changes have been added (for bug fixes / features)
* [ ] Docs have been added / updated (for bug fixes / features)
* [ ] The PR description links to the related issue (to link an issue, use '#XXX'.)
* [ ] The tests on AWS have been launched and are successful (comment with @slab-ci cpu_test and/or @slab-ci gpu_test to trigger the tests)
* [ ] The draft release description has been updated
* [ ] Check for breaking changes (including serialization changes) and add them to commit message following the conventional commit [specification][conventional-breaking]

[conventional-breaking]: https://www.conventionalcommits.org/en/v1.0.0/#commit-message-with-description-and-breaking-change-footer

### CODE REVIEW

**Comment by IceTDrinker** [comment]
@slab-ci cpu_test

File: concrete-core/src/backends/fftw/implementation/engines/fftw_engine/mod.rs [hunk 1]
```diff
@@ -30,6 +31,15 @@ impl Display for FftwError {
 
 impl Error for FftwError {}
 
+impl FftwError {
+    pub fn perform_generic_checks(polynomial_size: PolynomialSize) -> Result<(), Self> {
```
Comment (agnesLeroy) [inline concrete-core/src/backends/fftw/implementation/engines/fftw_engine/mod.rs:35]: Maybe I would name this function `perform_fftw_checks` instead.

**Review by agnesLeroy — APPROVED** [review]
Hello @IceTDrinker! Thanks a lot for this, totally makes sense. I just suggested another name for the function. :slightly_smiling_face:

**Comment by IceTDrinker** [comment]
@slab-ci cpu_test

**Pull request merged**